### DISCUSSION TITLE
请问有方法匹配父节点的最后一个子节点吗？

### DISCUSSION THREAD

**Discussion by Sunsetra** [gkd-kit/gkd#626] [Q&A] [answered]
现在我有一个节点结构树是这样的
![屏幕截图 2024-05-21 131126](https://github.com/gkd-kit/gkd/assets/16215245/43c48277-be83-46e1-8eed-9fdbae82b7a1)
请问有没有办法匹配“ViewGroup的末尾子节点，且类型是ImageView”？
类似`@ImageView[clickable=true] <-1 ViewGroup`这种，不用`<4`的原因是ViewGroup下子节点数量不定。

**Comment by adproqwq** [comment, marked as answer]
没有这种选择器，这种要写只能用其它特征定位

**Reply by Sunsetra** [reply]
想想也是，感谢解答
不过如果能有这种选择器会方便很多

**Comment by AIsouler** [comment]
可以试试```* <n * > 目标节点```

**Comment by Snoopy1866** [comment]
如果 `ImageView` 下没有子节点，可以使用 `[childCount=0]` 进行限定。

**Comment by 52comfortable** [comment]
最新的beta版本已经支持了

https://gkd.li/selector/node#int

ImageView[index=parent.childCount.minus(1)]

**Reply by Snoopy1866** [reply]
妙啊
